frontend/src/components/EditProfile/Profile.jsx

```diff
@@ -0,0 +1,88 @@
+import "../style/EditProfileStyles/EditProfile.scss";
+import React, { useState } from "react";
+import axios from "axios";
+
+const Profile = () => {
+  const [file, setFile] = useState("https://via.placeholder.com/250");
+  const [imagePreviewUrl, setImagePreviewUrl] = useState(
+    "https://via.placeholder.com/250"
+  );
+
+  const handleImageChange = e => {
+    e.preventDefault();
+    console.log(e.target.files);
+    console.log("----------------------");
+
+    let reader = new FileReader();
+    let selectedFile = e.target.files[0];
+
+    reader.onloadend = () => {
+      setFile(selectedFile);
+      setImagePreviewUrl(reader.result);
+    };
+
+    reader.readAsDataURL(selectedFile);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    // post of profile picture
+    let data = new FormData();
+    data.append("monfichier", file);
+    axios.post("http://localhost:5050/imgupload", data).then(response => {
+      alert("image added with success");
```

Comment:
I think this `alert` really breaks UX, and I'd prefer a quiet notification (if you really *need* to inform your user)